core/src/fnc/mod.rs

```diff
@@ -229,13 +229,13 @@ pub fn synchronous(ctx: &Context<'_>, name: &str, args: Vec<Value>) -> Result<Va
 		"rand::uuid::v7" => rand::uuid::v7,
 		"rand::uuid" => rand::uuid,
 		//
+		"session::ac" => session::ac(ctx),
 		"session::db" => session::db(ctx),
 		"session::id" => session::id(ctx),
 		"session::ip" => session::ip(ctx),
 		"session::ns" => session::ns(ctx),
 		"session::origin" => session::origin(ctx),
-		"session::sc" => session::sc(ctx),
-		"session::sd" => session::sd(ctx),
+		"session::rd" => session::rd(ctx),
```

Comment:
Do you mean to consider for cases where the session is associated with some data that is not specifically associated with a record? For now it would currently return the same data as `rd`, but it could eventually return any relevant session data. I can see value in implementing this now so that users don't have to migrate function calls if we eventually implement other "actors" that can own a session.

Reply:
I wouldn't bother, `$auth.*` seems fine to me. I didn't even know we had a `session:sd` function in the the first place!